features/step_definitions/individual_steps.rb

```diff
@@ -99,14 +99,16 @@ class VlpDocument
 end
 
 Then(/^Individual should see the error message Invalid Social Security number$/) do
-  expect(page.find("input[class='required floatlabel form-control interaction-field-control-person-ssn active-floatlabel']")[:oninvalid]).to eq "this.setCustomValidity('Invalid Social Security number.')"
+  wait_for_ajax
```

Comment:
Can we add a note here as to why we are making this wait call? Just for clarity in the future

Reply:
Along with every other this wait is used